h5p-bildetema/styles.css

```diff
@@ -21,10 +21,6 @@ BlankSlate is distributed under the terms of the GNU GPL
   font-family: sans-serif !important;
 }
 
-html {
-  scrollbar-gutter: stable;
-}
```

Comment:
Any particular reason why this was removed?

Reply:
This is only for the vite dev server. I removed it to match how it currently is in wordpress.